assets/js/shortcut-key.js

```diff
@@ -0,0 +1,13 @@
+// Check if the user is on a Mac and update the shortcut key for search accordingly
+(function () {
+  let isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+  document.addEventListener('readystatechange', function () {
+    if (document.readyState === 'interactive') {
```

Comment:
Is there any difference between doing this way and doing like this? Just out of curiosity:
```javascript
document.onreadystatechange = () => {
  if (document.readyState === 'complete') {
```

Reply:
Yes, if we wait until the document is fully loaded, you can consistently see the search change from `ctrl k` to `cmd k` in the navbar. Using `interactive` gives us access to manipulate the dom elements earlier in the document loading process as stated [here](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState)

Its not absolutely perfect.. meaning if you refresh over and over again you'll catch a glimpse of it changing maybe 1 out of 10 times, but its better than waiting until the readystate is complete

I think the way you have it though is cleaner. Updated 👍